AwsCryptographyPrimitives/codebuild/release-python/test-release.yml

```diff
@@ -0,0 +1,42 @@
+version: 0.2
+
+env:
+  variables:
+    BRANCH: "main"
+  secrets-manager:
+    TWINE_USERNAME: TestPyPiAPIToken:username
+    TWINE_PASSWORD: TestPyPiAPIToken:password
+
+phases:
+  install:
+    commands:
+      - pip install "tox < 4.0"
+      - pip install --upgrade pip
+    runtime-versions:
+      python: latest
+  pre_build:
+    commands:
+      - git checkout $COMMIT_ID
+      - FOUND_VERSION=$(sed -n 's/version = "\(.*\)"/\1/p' AwsCryptographyPrimitives/runtimes/python/pyproject.toml)
+      - |
+        if expr ${FOUND_VERSION} != ${VERSION}; then
+          echo "pyproject.toml version (${FOUND_VERSION}) does not match expected version (${VERSION}), stopping"
+          exit 1;
+        fi
+  build:
+    commands:
+      - cd AwsCryptographyPrimitives/runtimes/python/
+      - tox -e test-release
+
+batch:
+  fast-fail: true
+  build-graph:
+    - identifier: release_to_staging
```

Comment:
q; is this just an empty step? shouldn't the commands on line 27 be here instead?

Reply:
I'm not sure; this is lifted from ESDK-Python, which also doesn't specify this.
I'd expect that `release_to_staging` is the buildspec above?
But I would plan to confirm by testing